tensorboard/webapp/hparams/_redux/hparams_reducers_test.ts

```diff
@@ -630,4 +630,72 @@ describe('hparams/_redux/hparams_reducers_test', () => {
       expect(state2.dashboardSessionGroups).toEqual([mockSessionGroup]);
     });
   });
+
+  describe('dashboardHparamFilterAdded', () => {
+    it('adds entry dashboardFilters', () => {
```

Comment:
Can we add another test that contains existing items in `hparams` and ensures they are not overwritten?